velox/exec/WindowPartition.h

```diff
@@ -20,9 +20,16 @@
 
 /// Simple WindowPartition that builds over the RowContainer used for storing
 /// the input rows in the Window Operator. This works completely in-memory.
+/// WindowPartition supports partial window partitioning to facilitate
```

Comment:
@xiaoxmeng The WindowPartition is created by SortWindowBuild, PartitionStreamingWindowBuild, and RowsStreamingWindow. Since it has already been tested during the various window build tests, there is no need for separate testing of WindowPartition

Reply:
@xiaoxmeng : WindowPartition is accessed by all Window functions for the data. It is covered by all the unit tests already.